bindings/java/src/main/native/Makefile.am

```diff
@@ -5,7 +5,7 @@
 
 topdir=$(abs_top_builddir)
 javadir=$(top_srcdir)/bindings/java
-MVNCMD=$(MVN) -f $(javadir)/pom.xml
+MVNCMD=$(MVN) -B -f $(javadir)/pom.xml -Dmaven.repo.local=$(javadir)/.deps/
```

Comment:
why is the repo in srcdir and not builddir?

Reply:
Is there some variable to get builddir?